### DISCUSSION TITLE
Ideas

### DISCUSSION THREAD

**Discussion by fethivacation** [reworkd/AgentGPT#616] [Ideas]
إعطني مجموعة من الأفكار للتسويق الالكتروني

**Comment by Mostsanta** [comment]
I want to score a drop-down list of ready-made PROMPT blanks for various topics? Who's in favor ?

**Comment by valimikayilov** [comment]
why not to have a directory where agent gpt saves .pdfs, .imgs or any other files depending on the user's prompt

**Comment by jasangill1** [comment]
Hello, @Mostsanta @valimikayilov We actually have implemented these! Please try these features out and let us know what you think about them!!!! For no I will close this discussion. Please feel free to reopen this issue if you have any more suggestions, Alternativily you can always leave suggestions in our discord!! https://discord.gg/qNWFxxRJNh
